test/msnodesqlv8/msnodesqlv8.js

```diff
@@ -111,6 +111,7 @@ describe('msnodesqlv8', function () {
 
     it('bulk load (table)', done => TESTS['bulk load']('bulk_table', done))
     it.skip('bulk load (temporary table) (not supported by msnodesqlv8)', done => TESTS['bulk load']('#anohter_bulk_table', done))
```

Comment:
sorry to be a pain, but if this passes and it's using a temporary table, then maybe this could be unskipped?!

Reply:
Do we know if it will pass?

I can working on skipped tests later, but this says it's not supported so it's a bit of a scope creep change

Reply:
This PR is testing bulk loading with dates for msnodesql8; that test is not failing; so simply un-skipping to check if it is now supported should be an easy task, no?

Reply:
> This PR is testing bulk loading with dates for msnodesql8;

well, no it's testing it for tedious, because it doesn't (apparently) work on msnodesqlv8